setup.py

```diff
@@ -0,0 +1,144 @@
+
+    return Manage()
+
+
+class BuildWithParser(orig.build_py):
+    def initialize_options(self):
+        os.environ["GNATCOLL_ICONV_OPT"] = "-v"
+        manage_factory().run(
+            [
+                "--build-dir",
+                ".",
+                "make",
+                "--gargs",
+                f"-aP {base_dir}/contrib/gnatcoll-bindings/iconv "
+                f"-aP {base_dir}/contrib/gnatcoll-bindings/gmp",
+            ]
+        )
+        patch_rpath()
+        for l in [
+            "lib/librecordfluxdsllang/relocatable/dev/librecordfluxdsllang.so",
+            "lib/langkit_support/relocatable/dev/liblangkit_support.so",
+            f"{base_dir}/contrib/gnatcoll-bindings/iconv/lib/relocatable/libgnatcoll_iconv.so.0",
+            f"{base_dir}/contrib/gnatcoll-bindings/gmp/lib/relocatable/libgnatcoll_gmp.so.0",
+        ]:
+            source = Path(l)
+            shutil.copy(source, Path("python/librecordfluxdsllang/") / source.name)
+        super().initialize_options()
+
+
+class BuildParser(orig.build_py):
+    description = "Build RecordFlux langkit parser"
+
+    def run(self):
+        Path("build/langkit").mkdir(parents=True, exist_ok=True)
+        manage_factory().run(["--build-dir", "build/langkit", "--verbosity", "debug", "make"])
+
+
+with open("README.md") as f:
+    readme = f.read()
+
+setup(
+    name="RecordFlux-language",
+    version="0.1",
+    description=("RecordFlux language"),
+    long_description=readme,
+    long_description_content_type="text/markdown",
+    author="Alexander Senier",
+    author_email="senier@componolit.com",
+    url="https://github.com/Componolit/RecordFlux-language",
+    license="AGPL-3.0",
+    classifiers=[
+        "Development Status :: 5 - Production/Stable",
+        "Environment :: Console",
+        "License :: OSI Approved :: GNU Affero General Public License v3",
+        "Operating System :: POSIX :: Linux",
+        "Programming Language :: Ada",
+        "Programming Language :: Python :: 3 :: Only",
+        "Programming Language :: Python :: 3",
+        "Programming Language :: Python :: 3.7",
+        "Programming Language :: Python :: 3.8",
+        "Topic :: Communications",
+        "Topic :: Security",
+        "Topic :: Software Development :: Build Tools",
+        "Topic :: Software Development :: Code Generators",
+        "Topic :: System :: Networking",
+    ],
+    python_requires=">=3.7",
+    setup_requires=["langkit@" + LANGKIT],
+    dependency_links=[LANGKIT],
+    install_requires=["icontract >=2.3.4, <3"],
+    extras_require={
+        "devel": [
+            "black ==20.8b1",
+            "flake8 >=3, <4",
+            "hypothesis >=5, <6",
+            "isort >=5, <6",
+            "mypy >=0.770",
+            "pyicontract-lint >=2.0.0, <3",
+            "pylint >=2.6.0, <3",
+            "pytest >=5, <6",
+            "pytest-xdist >=1.32.0, <2",
+            "tqdm >=4, <5",
+            "langkit@" + LANGKIT,
```

Comment:
All dependencies which are not actively used should be removed.

Reply:
Changed in cd7456aa.